superflore/utils.py

```diff
@@ -202,52 +199,24 @@ def get_license(l):
     elif re.search(cc_re, l, f):
         return 'CC-BY-SA-3.0'
     elif re.search(boost_re, l, f):
-        return 'Boost-1.0'
+        return 'BSL-1.0'
```

Comment:
I'm not sure about this change... Gentoo likes this to be called `Boost-1.0`, so I would like it to remain as such. Unless there's some reason for it not to?

Reply:
Well, it looks like Gentoo is not following Linux Foundation SPDX standardization at https://spdx.org/licenses/ . Best would be Gentoo to use the standardized license. information.

I'll remove this from the patchset and submit separately.

Reply:
Yeah, sorry about that. You would have been flagged by this test, I think.

https://github.com/ros-infrastructure/superflore/blob/b2b928d5f5fa0bf9c9b458269a2c3306619cecb9/tests/test_utils.py#L100-L101

Reply:
Dropped patch updating boost license according to SPDX.